setup.py

```diff
@@ -51,6 +51,7 @@
         "numpy",
         "janus",
         "pyyaml",
+        "gputil",
```

Comment:
Add it to requirements.txt too.

Reply:
Forgot it, but you may found there is already a PR for this.